src/protocols/bdx/BdxMessages.cpp

```diff
@@ -99,17 +100,16 @@ CHIP_ERROR TransferInit::Parse(System::PacketBufferHandle aBuffer)
     uint32_t tmpUint32Value = 0; // Used for reading non-wide length and offset fields
     uint8_t * bufStart      = aBuffer->Start();
     Reader bufReader(bufStart, aBuffer->DataLength());
```

Comment:
I don't see any changes to `WriteToBuffer`. The new `mRangeCtlFlags` is public so it can be set by the caller. Is your plan just to ignore it when writing? If so, you should at least include a comment in the header about this